changelog.txt

```diff
@@ -1,5 +1,11 @@
 == Changelog ==
 
+* Tweak - Changed the word `Purchaser` to `Attendee` when email confirmation is sent for purchasing a ticket. [ETP-655]
```

Comment:
This changelog seems to be in the wrong place.

Reply:
Where should I put the changelog since this would be for a future release?

Reply:
Above the TBD version. But only on Readme.txt not on Changelog.txt